### PR TITLE
[ERMP] Task-3: Components part 1

### PR DESCRIPTION
**Pull request by AlexKarmaz**

### CODE REVIEW

File: src/components/Logo/Logo.jsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './Logo.css';
+
+const Logo = () => (
+    <>
+        <span className="logo"><b>netflix</b>roulette</span>
+    </>
+);
```
Comment (lvrubleuska) [inline src/components/Logo/Logo.jsx:8]: Is React Fragment needed here?

File: src/components/Search/Search.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './Search.css';
+
+const Search = () => (
+    <div className="searchWrapper">
+        <h1 className="searchTitle">FIND YOUR MOVIE</h1>
+        <div className="search">
+            <input className="searchInput" type="text" placeholder="What do you want to watch?"/>
+            <button className="searchButton">SEARCH</button>
```
Comment (lvrubleuska) [inline src/components/Search/Search.jsx:9]: You are using `text-transform: uppercase;` in filter and sorter, I would recommend using it for other cases as well

File: package.json [hunk 3]
```diff
@@ -24,10 +26,10 @@
...
     "cross-env": "^7.0.2",
     "css-loader": "^1.0.0",
-    "file-loader": "^6.0.0",
     "html-webpack-plugin": "^4.3.0",
     "jest": "^26.2.2",
     "mini-css-extract-plugin": "^0.9.0",
+    "prop-types": "^15.7.2",
     "react-hot-loader": "^4.12.21",
     "react-test-renderer": "^16.13.1",
     "style-loader": "^0.23.0",
```
Comment (lvrubleuska) [inline package.json:35]: I would consider configuring eslint

File: src/components/AddMovie/AddMovie.jsx [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import './AddMovie.css';
+
+const AddMovie = () => (
+    <button className="addMovieBtn">+ ADD MOVIE</button>
+);
+
+export default AddMovie
```
Comment (lvrubleuska) [inline src/components/AddMovie/AddMovie.jsx:8]: It is better to have newline at the end of files

**Review by lvrubleuska — APPROVED** [review]
Markup is done with React Components and React Fragments +
Apply styles +
Use PropTypes +
Use <ErrorBoundary> component for catching and displaying errors +

File: src/components/Header/Header.jsx [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Logo from "../Logo";
+import AddMovie from "../AddMovie"
+import Search from '../Search';
+import './Header.css';
+
```
[the remark this answers is not in the document]
Comment (AlexKarmaz) [reply src/components/Header/Header.jsx:6]: Yes, I agree. I will configure ESlint and Prettier.

File: src/components/Logo/Logo.jsx (hunk 1 above)
Comment (AlexKarmaz) [reply src/components/Logo/Logo.jsx:8]: Oo, it is redundant)

File: src/components/Search/Search.jsx (hunk 2 above)
Comment (AlexKarmaz) [reply src/components/Search/Search.jsx:9]: Will fix.

File: package.json (hunk 3 above)
Comment (AlexKarmaz) [reply package.json:35]: Will do. Thank.

File: src/components/AddMovie/AddMovie.jsx (hunk 4 above)
Comment (AlexKarmaz) [reply src/components/AddMovie/AddMovie.jsx:8]: will fix

**Pull request merged**